c_cxx/QNN_EP/mobilenetv2_classification/README.md

```diff
@@ -41,19 +45,21 @@
 - Windows 11
 - Visual Studio 2022
 - Python (needed to quantize model)
-- OnnxRuntime ARM Build with initial QNN support such as ONNX Runtime (ORT) Microsoft.ML.OnnxRuntime.QNN 1.15+ 
+- Qualcomm AI Engine Direct SDK (QNN SDK) from https://qpm.qualcomm.com/main/tools/details/qualcomm_ai_engine_direct
+    - Last known working QNN version: 2.14.1.230828
+- OnnxRuntime ARM Build with QNN support such as ONNX Runtime (ORT) Microsoft.ML.OnnxRuntime.QNN 1.17+ 
```

Comment:
Where do you find Microsoft.ML.OnnxRuntime.QNN 1.17+? I only see 1.16.1 as the latest available.

Reply:
It's the coming release. This example depend on a latest change in Onnxruntime. It works find if you build Onnxruntime from latest main branch. But if you choose Nuget package, you need to wait for the coming release.

---
In reply to: [1370304691](https://github.com/microsoft/onnxruntime-inference-examples/pull/324#discussion_r1370304691) [](ancestors = 1370304691)